src/Dapr.Client/DaprClient.cs

```diff
@@ -859,6 +859,39 @@ public async Task<StateEntry<TValue>> GetStateEntryAsync<TValue>(string storeNam
             string id,
             CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Get a response if the lock is acquired from the given lockstore. 
```

Comment:
I'd change this to:

`Attempt to lock the given resourceId with response indicating success.`

Provided that that is accurate. Does this wait for the lock to become available or does it fail immediately if the lock is already there?